com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/Camera/HDCameraFrameHistoryType.cs

```diff
@@ -49,6 +49,14 @@ public enum HDCameraFrameHistoryType
         VolumetricClouds1,
         /// <summary>Screen Space Reflection Accumulation.</summary>
         ScreenSpaceReflectionAccumulation,
+        /// <summary>Path traced Albedo AOV.</summary>
+        AlbedoAOV,
+        /// <summary>Path traced Normal AOV.</summary>
+        NormalAOV,
+        /// <summary>Path traced motion vecotr AOV.</summary>
```

Comment:
vecotr -->  vector

Reply:
Actually wait, these are just comments, not API doc, huh?

Reply:
Actually it's published as API doc in the package, so I'm fixing everything. Thanks for the comments.
(This is how we do scripting API docs in packages, hence the confusion in our meeting earlier today ;) )